modules/sandboxed-containers-building-blocks.adoc

```diff
@@ -4,4 +4,8 @@
 
 [id="sandboxed-containers-building-blocks_{context}"]
 
-= Sandboxed containers building blocks
+= {sandboxed-containers-first} building blocks
+
+The {sandboxed-containers-operator} encapsulates all of the pieces from Kata containers. The {sandboxed-containers-operator} manages installation, lifecycle, and configuration tasks.
+
+The {sandboxed-containers-operator} uses the link:https://quay.io/isolatedcontainers/sandboxed-containers-operator[sandboxed-containers-operator] container image. The image contains the controller part of the Operator that monitors and manages `KataConfig`.
```

Comment:
Is "uses:"

- "is based on"
- "consumes"
- something else

Reply:
@maxwelldb: The PM used "uses" here because that is the operator image. Maybe better to rephrase?

Reply:
Possibly. I was thinking of translation, here, but I'm not sure how much it matters, TBH.

Reply:
I'm just going to resolve this for now. I tried to rework the sentence but nothing I came up with made any sense >.<